### DISCUSSION TITLE
Errors while trying to install pyside2. pyside2 will not install.

### DISCUSSION THREAD

**Discussion by rleiman** [Homebrew/discussions#3191] [Linux]
Greetings,

On my MacBook, I tried to use this command as suggested online on several forums to install pyside2 but it responds with several errors.

`
brew install python3-pyside2.qt3dcore python3-pyside2.qt3dinput python3-pyside2.qt3dlogic python3-pyside2.qt3drender python3-pyside2.qtcharts python3-pyside2.qtconcurrent python3-pyside2.qtcore python3-pyside2.qtgui python3-pyside2.qthelp python3-pyside2.qtlocation python3-pyside2.qtmultimedia python3-pyside2.qtmultimediawidgets python3-pyside2.qtnetwork python3-pyside2.qtopengl python3-pyside2.qtpositioning python3-pyside2.qtprintsupport python3-pyside2.qtqml python3-pyside2.qtquick python3-pyside2.qtquickwidgets python3-pyside2.qtscript python3-pyside2.qtscripttools python3-pyside2.qtsensors python3-pyside2.qtsql python3-pyside2.qtsvg python3-pyside2.qttest python3-pyside2.qttexttospeech python3-pyside2.qtuitools python3-pyside2.qtwebchannel python3-pyside2.qtwebsockets python3-pyside2.qtwidgets python3-pyside2.qtx11extras python3-pyside2.qtxml python3-pyside2.qtxmlpatterns python3-pyside2uic

Warning: No available formula with the name "python3-pyside2.qt3dcore".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
`

All help will be appreciated.

Thanks.

**Comment by gromgit** [comment]
If you were following, say, [this SO question](https://stackoverflow.com/questions/69852228/how-to-install-pyside2-pyqt5-on-raspberry-pi-with-qtwebengine), note that it's for the Raspberry Pi's system package manager, _not_ for Homebrew.

Try `brew install pyside@2`. It may not give you all the components listed in your question, but it should at least cover the basics.
